lib/content/write.js

```diff
@@ -20,8 +20,7 @@ const writeFile = util.promisify(fs.writeFile)
 
 module.exports = write
 
-function write (cache, data, opts) {
-  opts = opts || {}
+function write (cache, data, opts = {}) {
   if (opts.algorithms && opts.algorithms.length > 1) {
```

Comment:
Would be better to do `const { integrity, algorithms } = opts` here, and avoid the `opts.<blah>` in this function.